packages/fluent-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -59,7 +59,7 @@ const SelectWidget = ({
   readonly,
   value,
   multiple,
-  autofocus,
+  // autofocus,
```

Comment:
Delete this maybe since it is not used... or perhaps you meant to uncomment?

Reply:
```suggestion
```